pkg/controller/aws.go

```diff
@@ -73,6 +73,7 @@ import (
 	"github.com/crossplane/provider-aws/pkg/controller/identity/iamuser"
 	"github.com/crossplane/provider-aws/pkg/controller/identity/iamuserpolicyattachment"
 	"github.com/crossplane/provider-aws/pkg/controller/identity/openidconnectprovider"
+	kafkaCluster "github.com/crossplane/provider-aws/pkg/controller/kafka/cluster"
```

Comment:
Perhaps just `kafkacluster` here? This alias is valid, but camel case package names aren't particularly idiomatic in Go. We've previously used package names like `dbcluster`, and `certificateauthority` for example.

https://blog.golang.org/package-names#TOC_2.

Reply:
done